myScript.js

```diff
@@ -197,3 +197,30 @@ function createProjectPopUp() {
 for (let i = 1; i <= 7; i += 1) {
   document.getElementById(`card${i.toString()}`).onclick = createProjectPopUp;
 }
+
+/// ////////Add Javascript form validator////////////////
+
+let form = null;
+
+if (window.innerWidth <= 768) {
+  form = document.getElementById('mobile-form');
+} else {
+  form = document.getElementById('desktop-form');
+}
+
+const { email } = form.elements;
+let message = null;
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const emailValue = email.value;
+
+  if (emailValue === emailValue.toLowerCase()) {
+    message = '';
+    form.submit();
+  } else {
+    message = 'please only enter lower case characters for email field';
+  }
+
+  form.querySelector('error').innerHTML = message;
+});
```

Comment:
When I click on the submit, it throws this error. Consider fixing it. 
Also, consider removing the value text in the message field. You can add placeholder text instead.
![image](https://user-images.githubusercontent.com/33268927/181469584-a689777a-1455-4b1b-a8ec-c1877dbaaee8.png)